Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -350,7 +350,7 @@ function vtkOpenGLTexture(publicAPI, model) {
 
   //----------------------------------------------------------------------------
   publicAPI.getInternalFormat = (vtktype, numComps) => {
-    if (!model.internalFormat) {
+    if (!model._forceInternalFormat) {
```

Comment:
I'm not sure if I fully follow the purpose of this. The only difference I see is that if `publicAPI.setInternalFormat(0)` is called, then we still set `_forceInternalFormat` to be true. The changes don't seem to affect much other than that.

Reply:
Without this change, `internalFormat` does not change between `create3DFilterableFromRaw` calls.  So if we change the component count of the input data, the `internalFormat` is not updated and there is an WebGL error.

Your right, hacky.  Maybe better to refactor this "getter" that sets and change lots of thing...

Reply:
Ah, I now understand what you are trying to do. @finetjul do you know the purpose of adding [this conditional in getInternalFormat](https://github.com/Kitware/vtk-js/pull/1400/files#diff-18c5d83bebc14325d84e520000b95e06dadffa088fe27eef138af64744171b89R316-R321)? I think that will help determine what functionality we should expect from calling `getInternalFormat`. Paul's proposed changes effectively revert that conditional, except when the user explicitly sets the internal format (which afaik nobody ever does).

Reply:
Without that conditional, earlier calls to publicAPI.setInternalFormat would get overwritten by all the `create*` functions' calls of `getInternalFormat`